index.html

```diff
@@ -56,6 +56,27 @@
         <img src="heart.svg" title="♥" style="max-width: clamp(1rem, 4rem + 10vw, 20rem); max-height: 20vh;">
         <h1 style="margin-block: 0;">Act Different.</h1>
       </hgroup>
+      <p>We, developers of software for your platform, ask that you begin to change course.</p>
+      <blockquote>
+        The people who are crazy enough<br>
+        to think they can change<br>
+        the world are the ones who do.<br>
+        <cite>&mdash; Steve Jobs @ Apple, 1997</cite>
+      </blockquote>
+      <h2>Our Vision for the&nbsp;Future</h2>
+      <ul>
+        <li>Removal of Affiliation with Twitter/X: @SwiftLang deletes their account on Twitter/X; no more Apple ads on the platform</li>
+      </ul>
+      <h2>Our Forms of&nbsp;Protest</h2>
+      <p>We don’t want to support an Apple that is this hostile. That’s why we, for example, &hellip;</p>
```

Comment:
This is a very vague way to say anything. I also feel like this doesn't properly say that any, not all of these points are part of the protest.

Maybe protest isn't even the right word. I let native English speakers decide that :) 

Suggestions very welcome!